source/server/http/admin.cc

```diff
@@ -163,7 +163,11 @@ Http::Code AdminImpl::handlerCpuProfiler(const std::string& url, Buffer::Instanc
 
   bool enable = query_params.begin()->second == "y";
   if (enable && !Profiler::Cpu::profilerEnabled()) {
-    Profiler::Cpu::startProfiler("/var/log/envoy/envoy.prof");
+    if (!Profiler::Cpu::startProfiler(profiler_path_)) {
+      response.add("?enable=<y|n>\n");
```

Comment:
I think a more meaningful response would indicate there was a failure to start the profiler.